### DISCUSSION TITLE
Pterodactyl Panel with in Proxmox

### DISCUSSION THREAD

**Discussion by ghost** [pterodactyl/panel#4524] [General Community]
Hello, I have a question is it possible to have Pterodactyl Panel create custom VMS with in Proxmox

**Comment by gOOvER** [comment]
No. Ptero is not for hosting VM's. Use Proxmox for this

**Comment by ghost** [comment]
I will be using Proxmox for creating VM'S I am just asking is it possible to have Pterodactyl Panel create the VM'S

**Comment by ghost** [comment]
like can I create a egg to do that

**Comment by gOOvER** [comment]
i dont think, that Ptero can't handle this. and an egg also cant handle this

**Comment by ghost** [comment]
Okay, Thanks for the help
